src/data/transformCityItem.js

```diff
@@ -0,0 +1,12 @@
+import Stub from '../assets/img/cities/Stub.png';
+
+const transformCityItem = city => {
+  return {
+    id: city.id,
+    name: city.name,
+    country: city.country.name,
```

Comment:
I would put optional chaining here in case BE is accidentally missing county for the city

```suggestion
    country: city.country?.name,
```